src/agents.py

```
@@ -1,125 +1,156 @@
-    )
\ No newline at end of file
+        goto=active_agent,
+    )
+
+
+# Tool responsável por delegar a criação dos nodes e egdes do sistema
+def supervisor_agent(
+    state: AgentState,
+) -> Command[list["node_creator"]]:
+    create_yaml_and_metadata("Sistema do usuario", " ")
+    novoState = DifyState = {
```

Ruff (0.8.2)

Local variable `DifyState` is assigned to but never used

Remove assignment to unused variable `DifyState`

(F841)